apps/flux/lib/flux/trades.ex

```diff
@@ -23,7 +23,7 @@ defmodule Flux.Trades do
     field(:acknowledged_at)
   end
 
-  def process_trade!(%EventBus.TradeExecuted{} = trade_params) do
```

Comment:
great. These dependencies should be dropped. This was to enforce that data structures used  are known by the EventBus. These validations will be the responsability of the EventBus.